.vscode/settings.json

```diff
@@ -0,0 +1,15 @@
+{
+  "editor.defaultFormatter": "biomejs.biome",
```

Comment:
not sure about biome vs prettier and eslint, but i guess for this project it's ok to try it out 🙂

Reply:
Nice thanks!